mlir/lib/Dialect/Tensor/IR/TensorOps.cpp

```diff
@@ -621,6 +621,18 @@ LogicalResult EmptyOp::verify() {
     return emitOpError("incorrect number of dynamic sizes, has ")
            << getDynamicSizes().size() << ", expected "
            << getType().getNumDynamicDims();
+
+  if (getDynamicSizes().size() > 0) {
+    if (llvm::any_of(getDynamicSizes(), [](Value operand) {
+          APInt constSizeArg;
+          if (!matchPattern(operand, m_ConstantInt(&constSizeArg))) {
```

Comment:
This does not belong to the verifier, because: https://mlir.llvm.org/getting_started/DeveloperGuide/#ir-verifier

Reply:
Thank you for teaching me about IR's verifier.

> This is why the guidelines to write verifier is to stick to information local to an operation (think “what I see when I print this operation alone”). Looking through operands or results is extremely unusual and should be avoided.

I understand it, so i will remove this checker from verifier.